include/linux/soundwire/sdw.h

```diff
@@ -547,6 +547,10 @@ struct sdw_slave_ops {
  * @node: node for bus list
  * @port_ready: Port ready completion flag for each Slave port
  * @dev_num: Device Number assigned by Bus
+ * @probed: boolean tracking driver state
+ * @probe_complete: completion utility to control potential races
+ * on startup between driver probe/initialization and SoundWire
+ * Slave state changes/imp-def interrupts
```

Comment:
I'm new to SDW. After I read the commit message, I think I can well understand what's the patch purpose and how the issue occurs. The only one is I can't know what "imp-def" is, even after grepping the whole soundwire code.

Reply:
This is a term used in the SoundWire spec, I will replace by implementation-defined.
That's precisely why I wanted feedback from everyone, thanks!